test/contracts/src/CallPermitDemo.sol

```diff
@@ -0,0 +1,125 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity >=0.8.3;
+
+import "precompiles/call-permit/CallPermit.sol";
```

Comment:
Have you tried loading it in Remix ? We used to have relative path to allow tools like VScode and Remix to access the file.
(same comment for all other solidity files using precompiles)